### DISCUSSION TITLE
The import android cannot be resolved

### DISCUSSION THREAD

**Discussion by eriksebastian88** [shreyashsaitwal/rush-cli#64] [Q&A]
Hi, nice to be here and try something new for me. 

several days ago I follow on youtube tutorial to build an extension. But I got some errors with VSCode while trying to build an extension 
![Screen Shot 2024-07-07 at 21 22 59](https://github.com/shreyashsaitwal/rush-cli/assets/30632841/0beed2d0-a71b-48a3-a47e-27669919d0fb)
and I am very new of this. would you help me to solve this? I do appreciate for your help.

**Comment by MahmoudHooda2019** [comment]
i think rush still not support VS code, you need to use IntelliJ IDEA Community Edition 
or waiting until @shreyashsaitwal upgrade to rush 2.0
